EmbeddedResourceVirtualPathProvider/EmbeddedResourceVirtualPathProvider.csproj

```diff
@@ -55,8 +55,8 @@
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <PropertyGroup>
-    <PostBuildEvent>del ..\..\..\nuget\lib\net40\*.* /F
-copy $(ProjectName).* ..\..\..\nuget\lib\net40\*.* /Y</PostBuildEvent>
+    <PostBuildEvent>del ..\..\..\nuget\EmbeddedResourceVirtualPathProvider.Core\lib\net40\*.* /F
+copy $(ProjectName).* ..\..\..\nuget\EmbeddedResourceVirtualPathProvider.Core\lib\net40\*.* /Y</PostBuildEvent>
```

Comment:
Should you  have left the original lines here too, so that the files get copied to the original folder as well?